server/ServerApp.cpp

```diff
@@ -1344,6 +1345,11 @@ void ServerApp::LoadGameInit(const std::vector<PlayerSaveGameData>& player_save_
             ErrorLogger() << "LoadGameInit got inconsistent empire ids between player save game data and result of PlayerEmpireID";
         }
 
+        // Revoke readiness only for online players so them can redo orders for the current turn.
+        // Without it server would advance turn because saves are made when all players sent orders
```

Comment:
Without doing this, the server would immediately advance the turn...

Reply:
Fixed.